walk-in-interview/src/main/webapp/business-jobs-list/script.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * This file is specific to show-job-posts-made.html
+ */
+
+// TODO(issue/21): get the language from the browser
+const CurrentLocale = 'en';
+
+/**
+ * Import statements are static so its parameters cannot be dynamic.
+ * TODO(issue/22): figure out how to use dynamic imports
+ */
+import {AppStrings} from '../strings.en.js';
+import {JOB_ID_PARAM, DEFAULT_PAGE_SIZE, DEFAULT_PAGE_INDEX,
+    setErrorMessage, getRequirementsList} from '../common-functions.js';
+import {API} from '../apis.js';
+
+const STRINGS = AppStrings['business-jobs-list'];
+const JOB_DETAILS_PATH = '../job-details/index.html';
+const HOMEPAGE_PATH = '../index.html';
+
+// TODO(issue/34): implement pagination for job listings
+
+window.onload = () => {
+  loadAndDisplayJobListings();
+}
+
+/**
+ * Add the list of jobs made.
+ */
+async function loadAndDisplayJobListings() {
+  const backButton = document.getElementById('back');
+  backButton.innerText = STRINGS['back'];
+  backButton.addEventListener('click', (_) => {
+    window.location.href = HOMEPAGE_PATH;
+  });
+
+  const jobListingsTitle =
+    document.getElementById('job-listings-title');
+  jobListingsTitle.innerText = STRINGS['job-listings-title'];
+
+  const jobPageData = 
+    await getJobsMade(DEFAULT_PAGE_SIZE, DEFAULT_PAGE_INDEX)
+      .catch((error) => {
+        console.error('error fetching job listings', error);
+        setErrorMessage(/* errorMessageElementId= */ 'error-message',
+            /* msg= */ STRINGS['get-jobs-error-message'],
+            /* include default msg= */ false);
+      });
+
+  displayJobListings(jobPageData);
+}
+
+/**
+ * Makes GET request to retrieve all the job posts made
+ * by the current business user. 
+ * This function is called when the interest page is loaded.
+ * 
+ * @param {int} pageSize The number of jobs for one page.
+ * @param {int} pageIndex The page index (starting from 0).
+ * @return {Object} The data returned from the servlet.
+ */
+function getJobsMade(pageSize, pageIndex) {
+  const params = `pageSize=${pageSize}&pageIndex=${pageIndex}`;
```

Comment:
nit: you should sanitize your inputs, can your convert pageSize and index to int before concat?